tests/providers/ssh/hooks/test_ssh.py

```diff
@@ -62,6 +69,7 @@ def tearDownClass(cls) -> None:
         with create_session() as session:
             conns_to_reset = [
                 cls.CONN_SSH_WITH_PRIVATE_KEY_EXTRA,
+                cls.CONN_SSH_WITH_PRIVATE_KEY_PASSPHRASE_EXTRA
```

Comment:
Hi @NadimYounes , may you add a comma at the end of this line? I'm not 100% sure if it's the reason, but worth a try.

I have to say the static check workflow is not working as what I'm expecting, because the error message is not displayed completely (to me).

Reply:
Agreed, the error message wasn't clear. You were spot on though, all checks passed, Cheers!